cmd/bumpversion.go

```diff
@@ -74,8 +74,9 @@ func upgradeGoFlutter(targetOS string, engineCachePath string) (err error) {
 	cmdGoGetU := exec.Command(build.GoBin(), "get", "-u", "github.com/go-flutter-desktop/go-flutter"+buildGoFlutterBranch)
 	cmdGoGetU.Dir = filepath.Join(wd, build.BuildPath)
 	cmdGoGetU.Env = append(os.Environ(),
-		"GOPROXY=direct", // github.com/golang/go/issues/32955 (allows '/' in branch name)
 		"GO111MODULE=on",
+		"GOPROXY=direct", // github.com/golang/go/issues/32955 (allows '/' in branch name)
+		"GOPRIVATE="+os.Getenv("GOPRIVATE"),
```

Comment:
I think this duplicates GOPRIVATE? On line 76 we already include the complete `os.Environ()`. This should already include GOPRIVATE. Same for the additions in the files below.

Having `GOPROXY=direct` does make sense, it overrides any value currently in the environment.